### PR TITLE
Fix --keep-app-running default and make devicelab verboser

### PR DESCRIPTION
**Pull request by Hixie**
cc @yjbanov

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/drive.dart [hunk 1]
```diff
@@ -41,6 +41,7 @@ class DriveCommand extends RunCommandBase {
   DriveCommand() {
     argParser.addFlag(
       'keep-app-running',
+      defaultsTo: null,
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/commands/drive.dart]: this is the line required to fix the build; the rest is just the debugging logic i added to track down the problem.

File: dev/devicelab/lib/framework/utils.dart [hunk 2]
```diff
@@ -184,8 +184,8 @@ Future<Process> startProcess(
   final ProcessInfo processInfo = new ProcessInfo(command, process);
   _runningProcesses.add(processInfo);
 
-  process.exitCode.whenComplete(() {
-    print('\n'); // separate the output of this script from subsequent output to make logs easier to read
+  process.exitCode.then((int exitCode) {
```
Comment (yjbanov) [inline dev/devicelab/lib/framework/utils.dart]: Not sure if it matters for `exitCode`, but the difference between `whenComplete` and `then` is that `whenComplete` will also trigger if there's an error.

**Review by yjbanov — APPROVED** [review]
LGTM

File: dev/devicelab/lib/framework/utils.dart (hunk 2 above)
Comment (Hixie) [inline dev/devicelab/lib/framework/utils.dart]: Yeah. exitCode won't ever fail though, and whenComplete doesn't give me the value.

**Comment by Hixie** [comment]
Landing on red to fix breakage.

**Pull request merged**